### PR TITLE
hosted update 2022-12-21

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
Pay attention to https://github.com/mendersoftware/mender-docs/pull/1972/files#diff-8cd4cdf145e16f784a8af79606c7b58db4b9b1a9e9ebbe38242476a7147d7e45 , 

Resolving the conflict there showed that current hosted version for this page does not have the header `Chapter 7` (see https://docs.mender.io/server-installation ), but I think that is a mistake from the past.

**Review by tranchitella — APPROVED** [review]

File: 06.Artifact-creation/10.CI-CD/docs.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+
+### Supported CI/CD providers
+
+Mender currently supports three major CI/CD providers:
+
+* [GitHub Actions](01.GitHub-Actions/docs.md)
+* [GitLab CI/CD](02.GitLab-CICD/docs.md)
+* [Azure DevOps](03.Azure-DevOps/docs.md)
+
```
Comment (TheMeaningfulEngineer) [inline 06.Artifact-creation/10.CI-CD/docs.md:31]: Not sure what to make of these scenarios.
Thought they were a 'high level overview of what can be integrated into the CI/CD system'.

But going to the actual Github/Gitlab/Azure examples they don't align with the scenarios.

File: 06.Artifact-creation/10.CI-CD/01.GitHub-Actions/docs.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## Pipelines examples
+
+* [Build and deploy Mender Artifact](#build-and-deploy-mender-artifact)
+* [Build and deploy multiple Mender Artifacts](#build-and-deploy-multiple-mender-artifacts)
+* [Build and deploy Mender Artifact to a single device](#build-and-deploy-mender-artifact-to-a-single-device)
+
+### Build and deploy Mender Artifact
+<!--AUTOVERSION: "tree/%/examples"/mender-ci-workflows-->
+```bash
+@build-and-deploy-mender-artifact.yml@ # https://github.com/mendersoftware/mender-ci-workflows/tree/1.0.0/examples/github/build-and-deploy-mender-artifact.yml
```
Comment (TheMeaningfulEngineer) [inline 06.Artifact-creation/10.CI-CD/01.GitHub-Actions/docs.md:35]: This doesn't render locally...
I guess that makes sense probably

File: 07.Server-installation/07.Device-license-count/docs.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+title: Device license count
+taxonomy:
+    category: docs
+    label: tutorial
+---
+
+!!!!! Device License Count is only available in Mender Enterprise.
```
Comment (TheMeaningfulEngineer) [inline 07.Server-installation/07.Device-license-count/docs.md:8]: Think this is an `Hosted Mender on-prem` feature only.
Not sure if it should even be in hosted.

File: 11.General/00.Hosted-Mender-regions/docs.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+title: Overview
+taxonomy:
+    category: docs
+---
+
```
Comment (TheMeaningfulEngineer) [inline 11.General/00.Hosted-Mender-regions/docs.md:6]: Shoaib was mentioning that the payment still all happen in dollars from the US Stripe instance.

That's something to add. (will ping him for a comment)

File: 06.Artifact-creation/05.Server-side-generation-of-Delta-Artifacts/docs.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+Mender Enterprise supports the server-side generation of Delta Artifacts.
+This document explains the prerequisites to enable this feature and how it works.
+
+!!! Hosted Mender supports this feature in the Enterprise plan; however, its availability requires a tenant-specific activation.
+!!! If you are a customer interested in it, please get in touch with the [Hosted Mender support team](mailto:support@mender.io).
+
+### Prerequisites
+
+To support the server-side generation of Delta Artifacts, your device must report the availability of the `mender-binary-delta` Update Module to the server.
```
[the remark this answers is not in the document]
Comment (lluiscampos) [reply 06.Artifact-creation/05.Server-side-generation-of-Delta-Artifacts/docs.md:16]: @tranchitella with assistance of @lluiscampos to follow-up.

File: 06.Artifact-creation/10.CI-CD/docs.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    category: docs
+---
+
+## CI/CD pipelines
+
+Mender provides support for application updates in the form of CI/CD pipelines to implement all kinds of specific use-cases. This section describes possible ways to integrate your workflow with Mender and most popular CI/CD providers.
+
+### Mender CI-workflows Docker image
+
+To help you in creating CI/CD pipelines, we created [GitHub repository](https://github.com/mendersoftware/mender-ci-workflows/) containing ready to use, battle-tested CI/CD pipeline code snippets that can be used in your workflows. Additionally, we provide [mender-ci-tools Docker image](https://hub.docker.com/r/mendersoftware/mender-ci-tools) based on Alpine Linux and providing command-line utilities,`mender-artifact` and `mender-cli` installed.
```
[the remark this answers is not in the document]
Comment (lluiscampos) [reply 06.Artifact-creation/10.CI-CD/docs.md:13]: I thought it was there already, actually

@lluiscampos to follow-up

File: 06.Artifact-creation/10.CI-CD/docs.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+To locally inspect the Docker image, pull it with the following command:
+<!--AUTOVERSION: "mendersoftware/mender-ci-tools:%"/mender-ci-workflows-->
+```
+docker pull mendersoftware/mender-ci-tools:1.0.0
+```
+
+The Docker image definition and CI/CD templates can be found [here](https://github.com/mendersoftware/mender-ci-workflows).
+
+
+### Supported CI/CD providers
```
[the remark this answers is not in the document]
Comment (lluiscampos) [reply 06.Artifact-creation/10.CI-CD/docs.md:24]: Means that we have tested them, as per criteria of the [MEN-5786](https://tracker.mender.io/browse/MEN-5786) epic. In some cases we have automated testing in others due to high complexity we have tested them manually.

File: 06.Artifact-creation/10.CI-CD/docs.md (hunk 1 above)
Comment (lluiscampos) [reply 06.Artifact-creation/10.CI-CD/docs.md:31]: @lluiscampos and @0lmi to follow-up.

File: 07.Server-installation/07.Device-license-count/docs.md (hunk 3 above)
Comment (lluiscampos) [reply 07.Server-installation/07.Device-license-count/docs.md:8]: @merlin-northern or @tranchitella to follow-up.

File: 11.General/00.Hosted-Mender-regions/docs.md (hunk 4 above)
Comment (lluiscampos) [reply 11.General/00.Hosted-Mender-regions/docs.md:6]: @tranchitella or @alfrunes to follow-up.

File: 06.Artifact-creation/10.CI-CD/01.GitHub-Actions/docs.md (hunk 2 above)
Comment (lluiscampos) [reply 06.Artifact-creation/10.CI-CD/01.GitHub-Actions/docs.md:35]: This is injected at build time. It was demoed couple of sprints ago. It should render locally with no problems if you are using latest `mender-docs-site` :+1:

**Pull request merged**